observability/opentelemetry/src/main/java/org/wildfly/extension/opentelemetry/OpenTelemetrySubsystemRegistrar.java

```diff
@@ -158,18 +169,71 @@ class OpenTelemetrySubsystemDefinition extends PersistentResourceDefinition {
             .setRestartAllServices()
             .build();
 
-    public static final AttributeDefinition[] ATTRIBUTES = {
+    public static final List<AttributeDefinition> ATTRIBUTES = List.of(
             SERVICE_NAME, EXPORTER, ENDPOINT, SPAN_PROCESSOR_TYPE, BATCH_DELAY, MAX_QUEUE_SIZE, MAX_EXPORT_BATCH_SIZE,
             EXPORT_TIMEOUT, SAMPLER, RATIO
-    };
+    );
+
+    private final AtomicReference<WildFlyOpenTelemetryConfig> opentelemetryConfig = new AtomicReference<>();
+
+    @Override
+    public ManagementResourceRegistration register(SubsystemRegistration parent,
+                                                   ManagementResourceRegistrationContext context) {
+        ManagementResourceRegistration registration =
+                parent.registerSubsystemModel(ResourceDefinition.builder(ResourceRegistration.of(SUBSYSTEM_PATH),
+                        SUBSYSTEM_RESOLVER).build());
+        ResourceDescriptor descriptor = ResourceDescriptor.builder(SUBSYSTEM_RESOLVER)
+                .addCapability(OPENTELEMETRY_CAPABILITY)
+                .withRuntimeHandler(ResourceOperationRuntimeHandler.configureService(this))
+                .addAttributes(ATTRIBUTES)
+                .withAddOperationRestartFlag(OperationEntry.Flag.RESTART_ALL_SERVICES)
+                .withRemoveOperationRestartFlag(OperationEntry.Flag.RESTART_ALL_SERVICES)
+                .withDeploymentChainContributor(target -> {
+                    // We need to disable vertx's DNS resolver as it causes failures under k8s
+                    if (System.getProperty(VERTX_DISABLE_DNS_RESOLVER) == null) {
+                        System.setProperty(VERTX_DISABLE_DNS_RESOLVER, "true");
+                    }
+                    target.addDeploymentProcessor(OpenTelemetryConfigurationConstants.SUBSYSTEM_NAME,
+                            Phase.DEPENDENCIES,
+                            0x1910,
```

Comment:
This value should reside in Phase.java within wildfly-core.

Reply:
https://github.com/wildfly/wildfly-core/pull/6075